index.js

```diff
@@ -135,6 +134,12 @@ const COMPRESSIONS = {
 class Cache {
   constructor(key, _) {
     const options = _ || {};
+    let tmpdir = path.join(os.tmpdir(), "async-disk-user");
+    try {
+      tmpdir = path.join(os.tmpdir(), username());
```

Comment:
Running both `tmpdir()` and `username()` for each `Cache` created may slow down cache creation.

These values are not expected to change once a process starts, which is why they where originally derived in module scope above. Was that change needed?